rust/src/arrow/linalg.rs

```diff
@@ -447,13 +444,18 @@ mod tests {
             -0.6525516,
             0.10910681,
         ];
-        assert_relative_eq!(u.data().values(), expected_u.as_slice(), epsilon = 0.0001,);
-
-        assert_relative_eq!(
-            sigma.values(),
-            vec![27.46873242, 22.64318501, 8.55838823, 5.9857232, 2.01489966].as_slice(),
-            epsilon = 0.0001,
-        );
+        u.data()
```

Comment:
why can we just use slice comparision here?

Reply:
complains about ScalarBuffe<f32> not having AbsDiffEq